packages/edit-navigation/docs/user-documentation.md

```diff
@@ -0,0 +1,72 @@
+Note: this documentation is a work in progress.
+
+# Block-based menu editor
+
+The Block-based Navigation Editor brings the power of blocks to the Appearance > Navigation section in the WordPress Administration Screens allowing you use blocks to create menus.
+
+## How to use the editor
+
+The interface replicates the Post Editor experience, allowing you to use similar workflows like drag and drop.
+
+The editor provides access to all the menus created using the previous Menu screen in WordPress.
+
+New menus can also easily be created, providing options to easily create a menu from an existing page hierarchy, by copying an existing menu, or creating a blank menu.
+
+Rather than the paragraph being the go-to block, the Navigation Editor provides a link block for creating links to different types of items. Page, post, tag, category and custom links can all be created. Links can be nested to create sub-menus.
+
+### How to create a menu
+
+1. At the top of the editor select 'New menu'.
+2. Type the name of your menu.
+3. Choose a starting option of either 'Start blank', 'Add all pages', or 'Copy existing menu'.
+
+### How to add a block
+
+1. Click one of the `+` buttons.
+2. Select the type of link you want to add.
+3. Use the dialog to search for the item you want to link to. For a custom link, type in a URL and hit 'Enter'.
```

Comment:
```suggestion
3. One the item is inserted into the editor, use the dialog to search for the item you want to link to. Alternatively, for a custom link, type in a URL and hit 'Enter'.
```

Reply:
I don't think 'Once the item is inserted into the editor' is needed because the line above tells them to insert it and it happens immediately.

I have changed this by splitting out the instructions for a custom link, which I think makes it clearer.